deployment/quick-start/services-configuration.yaml.template

```diff
@@ -15,131 +15,110 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-cluster:
 
-  clusterid: pai-example
 
-  # HDFS, zookeeper data path on your cluster machine.
-  data-path: "/datastorage"
+cluster:
+  common:
+    clusterid: pai-example
+
+    # HDFS, zookeeper data path on your cluster machine.
+    data-path: "/datastorage"
```

Comment:
Is this field mandatory? I see it is commented out in example. Should go through all fields in this file, and comment out all optional fields. Also I think we should refine some of comments, so this file can server as document of how to configure pai.

Reply:
I will comment the ```common``` field. But the ```docker-registry``` is necessary, because when doing ci/cd, we will fill some value into this key.